test/rulesets/Base/arraymath.jl

```diff
@@ -15,12 +15,27 @@
         ⋆₂(a, b) = ⋆₂((a, b))  #matrix
         ⋆₂() = ⋆₂(())  # scalar
 
+        ⋆₃(a) = (⋆(n), ⋆(n, 1))
```

Comment:
I don't understand what this is for?
`⋆` makes a nice random value
`⋆₂` makes a nice random value and its cotangent